api_bridge_agent_demos/mcp/README.md

```diff
@@ -0,0 +1,111 @@
+# API Bridge Agent MCP demo
+
+This application demonstrates the use of MCP (Model Context Protocol) in API
+Bridge Agent.
+
+The role of this application is to take a Github repository as input, analyze
+the repository and then look for related contents on DuckDuckGo search engine.
+
+The setup consists of:
+- An application developed with the LlamaIndex framework.
+- API Bridge Agent configured with at least the Github API, and the DuckDuckGo
+MCP server.
+
+# Configuration
+
```

Comment:
Can you add a prerequisites section with things like uv, docker, and the api-bridge, I guess?